test/test.js

```diff
@@ -37,4 +37,14 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {
+      detectImageSize: true,
+      resolveImagePath: (value, source) => {
+        expect(typeof value).toBe('string');
+        expect(source).toHaveProperty('input.css');
+        return path.resolve(process.cwd(), value);
+      }}, done);
+  });
```

Comment:
Perhaps something like this?:

```js
    const spy = jest.fn();
    runTest('resolve-image-path.css', {
      detectImageSize: true,
      resolveImagePath: spy,
    }, done);
    const spyCall = spy.mock.calls[0];
    expect(spyCall[0]).toEqual('./test/fixtures/images/cat.jpg');
    expect(spyCall[1]).toMatchSnapshot();
```
The `spyCall` is an array of the arguments passed to the fake function.